### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by maxmouchet**
À faire :
- [x] Compléter le README.md (_Features_)
- [x] Finir la section _Quick Start_ de la documentation
- [x] Documenter l'utilisation des _properties_ (`GraphQLAuthService(...)`)
- [x] Rename DOCS_TOKEN to GITHUB_PAT
- [x] Simplifier l'exemple de requête GraphQL
- [x] (@maxmouchet) Publier le dépôt `sphinx-plugins` et changer le chemin dans `conf.py`
- [x] (@jrebecchi) Créer un [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) avec le permission `repo`, et le mettre dans le secret `GITHUB_PAT` du dépôt.

### CODE REVIEW

**Comment by maxmouchet** [comment]
@jrebecchi mets tes idées de features pour le README en commentaire.

**Comment by jrebecchi** [comment]
Thanks, Max! 
I just added the GITHUB_PAT token to the repo. 
Find bellow a list of possible features for GraphQL Auth Service:

- Authentication based on JSONWebTokens
- XSS and CSRF protection
- Shape it with your own User Data Model in 1 sec.
- Easy to integrate into web apps / mobile apps
- Easy to scale
- Test IDE included for development

File: docs/quick-start.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                'Content-Type': 'application/json',
+                // Authentication token to make an authenticated request
+                'Authorization': 'Bearer ' + yourAuthToken
+            },
+            body: JSON.stringify({
+                query: query
+            }),
+        })
+        .then(res => res.json())
+        .then(res => console.log(res));
```
Comment (jrebecchi) [inline docs/quick-start.rst]: let headers = {
    'Content-Type': 'application/json',
    'Authorization': 'Bearer ' + yourAuthToken // To make an authenticated request
};
let query =
    `mutation {
        updateMe(fields: {username:"newusername"}) {
            notifications{
                message
            }
        }
    }`;
let body = JSON.stringify({ query });

fetch('http://your-service-url.com', { method: 'post', headers, body })
.then(res => res.json())
.then(res => console.log(res));

File: README.md [hunk 2]
```diff
@@ -1,78 +1,38 @@
...
 
 ## Features
-  * Registration
-  * Login
-  * Email verification
-  * Password recovery
-  * Account modification
-  * Account deletion
-  * API to fetch public user info
-  * [XSS](https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)) and [CSRF](https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF)) protection
```
Comment (jrebecchi) [inline README.md]: - Authentication based on JSONWebTokens
- XSS and CSRF protection
- Shape it with your own User Data Model in 1 sec.
- Easy to integrate into web apps / mobile apps
- Easy to scale
- Test IDE included for development

**Comment by maxmouchet** [comment]
Ready for merge on my side. Proof-reading and further updates can be applied later on.

**Comment by jrebecchi** [comment]
Thanks for the PR Max! Great new milestone for the project on which will be based further updates.

I do this week a deep review.

**Pull request merged**